source/core/Array.cpp

```diff
@@ -1201,15 +1201,31 @@ VIREO_FUNCTION_SIGNATURET(ArrayMaxMinInternal, FindArrayMaxMinInstruction)
 }
```

Comment:
@amehra-ni Can you update the broken links in the docs that are failing the build? https://github.com/ni/VireoSDK/runs/6856533744?check_suite_focus=true#step:8:152

It looks GitHub just moved them, if you visit the links in a browser you get redirected to the new locations ie

https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/getting-started/about-collaborative-development-models
https://docs.github.com/en/get-started/quickstart/fork-a-repo
https://docs.github.com/en/get-started/quickstart/fork-a-repo#cloning-your-forked-repository
https://docs.github.com/en/get-started/quickstart/fork-a-repo#configuring-git-to-sync-your-fork-with-the-original-repository

Reply:
I pushed a change to main that you can merge to disable link checking for the github docs and created the following issue: https://github.com/ni/VireoSDK/issues/701